server/api/serializers.py

```diff
@@ -1,12 +1,39 @@
 from django.contrib.auth.models import User, Group
 from rest_framework import serializers
+from rest_framework_jwt.settings import api_settings
 from .models import *
 
 # Serializers to support seralizing User and Group objects
 class UserSerializer(serializers.HyperlinkedModelSerializer):
     class Meta:
         model = User
-        fields = ['email', 'first_name', 'last_name', 'groups', 'is_staff']
+        # Should it have the 'username' field?
+        fields = ['email', 'first_name', 'last_name', 'groups', 'is_staff', 'username']
```

Comment:
I don't think we necessarily need username in addition to email/first/last, but up to you

Reply:
I think the "username" field is required?